tests/integration/parity/test_parity_http.py

```diff
@@ -49,6 +49,7 @@ def parity_command_arguments(
         '--password', passwordfile,
         '--jsonrpc-port', rpc_port,
         '--jsonrpc-apis', 'all',
+        '--jsonrpc-experimental',
```

Comment:
I'm confused,  Tests above suggest that new APIs are not in parity.  This line suggests it enables them (or enables something...???)

Reply:
IIRC, personal_signTypedData is implemented in parity, but eth_signTypedData is not yet. It looks like Geth has them merged into master, but they haven't been released yet.